src/frontend/src/flows/manage/index.ts

```diff
@@ -132,6 +133,11 @@ const displayManageTemplate = ({
       </p>
     </hgroup>
     ${anchorSection(userNumber)}
+    <p class="t-paragraph">
```

Comment:
in general you can use `l-stack? (on the button) a `p` should only be used for actual paragraphs. But since this will probably be replaced its definitely not blocking

Reply:
I thought I'd make it as ugly as possible so that I make sure you look at everything carefully when you add the design 😄  


but good to know, thanks!